docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md

```diff
@@ -0,0 +1,208 @@
+
+  > **All PMK cluster nodes would have kubectl installed once cluster is deployed.**
+
+* [Azure CLI (az)](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) version 2.15.0 and above. This cli tool would help to setup the Azure resources and connect the Kubernetes cluster to Azure Arc.
+
+* [Helm](https://helm.sh/docs/intro/install/) version 3+ , to install the Azure Arc agents on the cluster.
+
+* [Enable subscription](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider) with the two resource providers for Azure Arc-enabled Kubernetes.
+
+  ```shell
+  az provider register --namespace Microsoft.Kubernetes
+  az provider register --namespace Microsoft.KubernetesConfiguration
+  az provider register --namespace Microsoft.ExtendedLocation
+  ```
+
+  Registration is an asynchronous process, and registration may take approximately 10 minutes. You can monitor the registration process with the following commands
+
+  ```shell
+  az provider show -n Microsoft.Kubernetes -o table
+  az provider show -n Microsoft.KubernetesConfiguration -o table
+  az provider show -n Microsoft.ExtendedLocation -o table
+  ```
+
+* Install the Azure Arc Kubernetes CLI extensions *connectedk8s* and *k8s-configuration* .
+
+  ```shell
+  az extension add --name connectedk8s
+  az extension add --name k8s-configuration
+  ```
+
+  Run below command to verify that it is installed.
+
+  ```shell
+  az extension list -o table
+  ```
+
+* Create Azure service principal (SP)
+  To be able to complete the scenario and its related automation, Azure service principal assigned with the *Contributor* role is required.
+
+  For creating the service principal, first login to your Azure account.
+
+  ```shell
+  az login
+  ```
+
+  Login by opening the [Microsoft devicelogin page](https://microsoft.com/devicelogin) on browser and enter the code to authenticate.
+
+  An example output of successful authentication is shown below;
+
+  ![MSFT Device Login](./01.png)
+
+  Once you have a successful login, create the Azure service principal (SP).
+
+  ```shell
+  az ad sp create-for-rbac -n "<service-principal-name>" --role contributor
+  ```
+
+  See below example of creating a service principal.
+
+  ![Create Service Principal](./02.png)
+
+  > *Note : It is highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).*
+
+* Create an Azure resource group where you want the PMK cluster to show up.
+
+  ```shell
+  az group create -l <Azure Region> -n <resource group name>
+  ```
+
+  See below example of creating a resource group.
```

Comment:
```suggestion
  See below example of creating a resource group.
```

No need for an extra line here. It would also be good to add a screenshot of the empty resource group in the Azure portal (don't forget to mask subscription id).